### PR TITLE
log_in

### PR DESCRIPTION
**Pull request by tenyiting**
Ready

### CODE REVIEW

**Comment by tenyiting** [comment]
Ready

File: app/helpers/sessions_helper.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  def logged_in(user)
+    session[:user_id] = user.id
+  end
+
+  # Returns the current logged-in user (if any).
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def current_user
```
Comment (ngocvt-0484) [inline app/helpers/sessions_helper.rb:12]: duplicate function
you defined it on line 8

**Comment by tenyiting** [comment]
Ready

File: app/assets/stylesheets/sessions.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (huongnt-2545) [inline app/assets/stylesheets/sessions.scss:1]: delete this file

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -1,11 +1,31 @@
 <header class="navbar navbar-fixed-top navbar-inverse">
   <div class="container">
-    <%= link_to t("sample_app"), "#", id: "logo" %>
+    <%= link_to t(:title) , "#", id: "logo" %>
     <nav>
       <ul class="nav navbar-nav navbar-right">
-        <li><%= link_to t(:Home), "#" %></li>
-        <li><%= link_to t(:Help), "#" %></li>
-        <li><%= link_to t(:Log_in), "#" %></li>
+      <li><%= link_to t(:Home), home_path %></li>
```
Comment (huongnt-2545) [inline app/views/layouts/_header.html.erb:6]: wrong indent from line 6 -> line 28

File: app/views/sessions/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<% provide :title, t("log_in") %>
+<h1><%= t("log_in") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+      <%= form_for :session, url: login_path do |f| %>
```
Comment (huongnt-2545) [inline app/views/sessions/new.html.erb:5]: wrong indent
```suggestion
    <%= form_for :session, url: login_path do |f| %>
```

File: test/controllers/sessions_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/controllers/sessions_controller_test.rb:1]: delete this file

**Comment by tenyiting** [comment]
ready

**Pull request merged**